app/models/request.rb

```diff
@@ -37,9 +38,9 @@ def personalized_text(contributor)
   def stats
     {
       counts: {
-        recipients: messages.map(&:recipient_id).compact.uniq.size,
-        contributors: messages.select(&:reply?).map(&:sender_id).compact.uniq.size,
-        photos: replies.pluck(:photos_count).sum,
+        recipients: outbound.select(:recipient_id).distinct.count,
+        contributors: replies.select(:sender_id).distinct.count,
+        photos: replies.sum(:photos_count),
```

Comment:
Favor returning as little as possible from the db and handling count/sum with postgres instead of ruby.